client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/PutDataFrameTransformRequest.java

```diff
@@ -41,6 +43,19 @@ public DataFrameTransformConfig getConfig() {
         return config;
     }
 
+    public Boolean getDeferValidations() {
```

Comment:
Why is it sometimes spelled singular ("validation") and sometimes plural ("validations")? Is it deliberate?

Reply:
I'm +1 for the singular `deferValidation`